src/app/tasks/routes.py

```
@@ -195,49 +207,51 @@ def start():
-                form=request.form.to_dict(flat=True)
+                form=request.form.to_dict(flat=True),
             )
         except TaskAlreadyExistsError as exc:
             existing = exc.task
-            return redirect(url_for("main.task1", task_id=existing["id"], title=title, error="task-active"))
-        except Exception:
-            logger.exception("Failed to create task")
+            return redirect(
+                url_for("main.task1", task_id=existing["id"], title=title, error="task-active")
+            )
+        except Exception as exc:  # noqa: BLE001
```

Ruff (0.14.1)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)